src/app/health/health.component.html

```diff
@@ -7,6 +7,8 @@
   <mat-toolbar class="primary-color font-size-1">
     <a mat-raised-button color="accent" class="margin-lr-3" i18n [routerLink]="['update']">Update Details</a>
     <a mat-raised-button color="accent" class="margin-lr-3" i18n [routerLink]="['event', { id: userDetail._id }]">Add Examination</a>
+    <span class="toolbar-fill"></span>
+    <a mat-raised-button color="primary" class="margin-lr-3" i18n (click)="exportCSV()" >Export Examination</a>
```

Comment:
This should be the accent color or a mat-stroked-button so it does not blend into the toolbar